### PR TITLE
Endpoint standardization

### PR DESCRIPTION
**Pull request by davidjovanovic**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ It will expose the following endpoints (both over https):
 ```
 * /incoming-data-app/multipartMessageBodyBinary to receive data (MultiPartMessage) with binary body from Data App (the A endpoint in the above picture)
 * /incoming-data-app/multipartMessageBodyFormData to receive data (MultiPartMessage) with form-data body from Data App (the A endpoint in the above picture)
```
Comment (IgorBalog-Eng) [inline README.md:31]: Can you add also httpHeader A-endpoint here also, looks like we forgot to add it in README :(

File: README.md [hunk 2]
```diff
@@ -59,7 +59,7 @@ The sender DataApp should send a request using the following schema, specifing i
 ```
 curl -k -request POST 'https://{IPADDRESS}:{A_ENDPOINT_PUBLIC_PORT}/incoming-data-app/multipartMessageBodyBinary' 
    --header 'Content-Type: multipart/mixed; boundary=CQWZRdCCXr5aIuonjmRXF-QzcZ2Kyi4Dkn6' 
-   --header 'Forward-To: {RECEIVER_IP_ADDRESS}:{B_ENDPOINT_PUBLIC_PORT}/incoming-data-channel/receivedMessage' 
+   --header 'Forward-To: {RECEIVER_IP_ADDRESS}:{B_ENDPOINT_PUBLIC_PORT}/data'
```
Comment (IgorBalog-Eng) [inline README.md:62]: Good catch to update README file in general 👍

File: README.md (hunk 1 above)
Comment (davidjovanovic) [reply README.md:31]: On my way :)

File: README.md (hunk 2 above)
Comment (davidjovanovic) [reply README.md:63]: Thanks 👍

**Comment by IgorBalog-Eng** [comment]
Merged to master

**Pull request merged**